pkg/apis/serving/v1alpha2/framework_tensorrt.go

```diff
@@ -38,7 +38,7 @@ func (t *TensorRTSpec) GetResourceRequirements() *v1.ResourceRequirements {
 	return &t.Resources
 }
 
-func (t *TensorRTSpec) GetContainer(modelName string, config *InferenceServicesConfig) *v1.Container {
+func (t *TensorRTSpec) GetContainer(modelName string, parallelism int, config *InferenceServicesConfig) *v1.Container {
```

Comment:
As per tfserving should parallism be validated for TRTIS as it doesn't apply to this server?

Reply:
As above, parallelism changes the target concurrency for all the frameworks,  for framework that uses tornado servers it sets the number of workers, since tfserving and trtis has dynamic thread pool we do not do much there.